test/gerkin/features/steps/data_list.js

```diff
@@ -0,0 +1,97 @@
+const { spec, expect, stores } = require('pactum');
+const { When, Then, Given, After } = require('@cucumber/cucumber');
+
+const firstSearchedName = 'John';
+const secondSearchedName = 'Adrien';
+const baseUrl = 'http://localhost:3333/data/registry1/version1';
+const header = {
+  key: 'Information-Mediator-Client',
+  value: 'INSTANCE/CLASS/MEMBER/SUBSYSTEM',
+};
+
+// scenario 1
+Given('The user wants to search for "John Helmut" in the database', () => {
+  return firstSearchedName;
```

Comment:
it would be better to set the searched user name here.

So at the top of the file leave the definition empty:
`let firstSearchedName;`

And here set it to 'John Helmut'
`firstSearchedName = 'John Helmut'`